src/Filesystem/Folder.php

```diff
@@ -194,16 +194,32 @@ public function read($sort = true, $exceptions = false, $fullPath = false)
             if ($fullPath) {
                 $name = $item->getPathname();
             }
+
             if ($item->isDir()) {
-                $dirs[] = $name;
+                $dirs[$item->getCTime()][] = $name;
             } else {
-                $files[] = $name;
+                $files[$item->getCTime()][] = $name;
```

Comment:
@Tzaoh I am not sure that you get what I mean. Sorry if you have already got it. I wanted to say, the `sort($files)` seems not to work correctly. Isn't `$files` a 2d array?
```php
$files = [
    100 => [ 'a.txt', 'c.txt' ],
    101 => [ 'b.txt', ],
]
```